src/main.cpp

```diff
@@ -1,8 +1,17 @@
+#include <iostream>
+#include <stdlib.h>
 #include "Dollar.h"
- 
-int main(int argc, char** argv) {
 
-	Dollar *dollar = new Dollar();	
+int main(int argc, char** argv) {
+	
+	if(argc < 3){
+		std::cout << "Use: ./runDollar <amount> <multiplier>" << std::endl;
+		return EXIT_SUCCESS;
+	}	
+	Dollar *dollar = new Dollar(atoi(argv[1]));
```

Comment:
I think that is this case you might want to avoid using pointers by declaring Dollar in the stack.